hack/tools/install_vault.sh

```diff
@@ -6,7 +6,7 @@
 cd "${0%/*}"
 source ./common.sh
 
-version=1.4.3
```

Comment:
Hi @jtcheng , if you're changing this file, please add the following:

```shell
DESIRED_VERSION=1.8.9

header_text "Checking for bin/vault $DESIRED_VERSION"
[[ -f bin/vault &&  `bin/vault -v | awk '{print $2}'` == "v$DESIRED_VERSION" ]] && exit 0

header_text "Installing bin/vault $DESIRED_VERSION"
mkdir -p ./bin
curl -L -o ./bin/vault.zip https://releases.hashicorp.com/vault/${DESIRED_VERSION}/vault_${DESIRED_VERSION}_${os}_amd64.zip
unzip -o ./bin/vault.zip -d ./bin/
rm ./bin/vault.zip
```

Pls. put attenshion on the `-o` flag of unzip

Reply:
@roytman 
Thanks for the comments.
This changes make sure we can use the desired version of local binary.
The only works when we directly run `./install_vault.sh`.
`make install-tools` will not works, since it will exit earlier as the binary target already met.